### PR TITLE
add infrahub

### PR DESCRIPTION
**Pull request by petercrocker**
Added Infrahub open-source project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -252,6 +252,15 @@ Network Automation is a cross between the discipline of [Network Infrastructure]
 
  - [Using Chef Client with Cisco NX-OS](https://www.cisco.com/c/en/us/td/docs/switches/datacenter/nexus9000/sw/7-x/programmability/guide/b_Cisco_Nexus_9000_Series_NX-OS_Programmability_Guide_7x/b_Cisco_Nexus_9000_Series_NX-OS_Programmability_Guide_7x_chapter_01110.html) - Using Chef Client with Cisco NX-OS
 
+## Infrahub
+
+- [Infrahub GitHub Repo](https://github.com/opsmill/infrahub) - Infrahub offers a central hub to manage the data, templates and playbooks that powers your infrastructure by combining the version control and branch management capabilities of Git with the flexible data model and UI of a graph database.
```
Comment (itdependsnetworks) [inline README.md:257]: This is generally a copy/paste from the "about" on github, which currently is "Infrahub - A new approach to Infrastructure Management", can you have these two match?

File: README.md [hunk 2]
```diff
@@ -252,6 +252,15 @@ Network Automation is a cross between the discipline of [Network Infrastructure]
...
+- Getting Started with Infrahub
```
Comment (itdependsnetworks) [inline README.md:258]: Let's just do the docs page here, there should be a full sentence ending with a period as a description.

Comment (petercrocker) [reply README.md:258]: Simplified, let me know if this is better!

File: README.md (hunk 1 above)
Comment (petercrocker) [reply README.md:257]: I was trying to keep the contents in this description short and to the point, inline with the amount of text on all other listings in this README, where the "about" section on github can be more verbose. "A new approach" doesn't actually say what the project does. So not quite sure how to reconcile this?

File: README.md [hunk 3]
```diff
@@ -252,6 +252,12 @@ Network Automation is a cross between the discipline of [Network Infrastructure]
...
+  - [Infrahub Documentation](https://docs.infrahub.app/) - Including overview, getting started and FAQ.
```
Comment (itdependsnetworks) [inline README.md:259]:
```suggestion
- [Infrahub GitHub Repo](https://github.com/opsmill/infrahub) - Infrahub - A new approach to Infrastructure Management.
- [Infrahub Documentation](https://docs.infrahub.app/) - Including overview, getting started, and FAQ.
```

**Review by itdependsnetworks — APPROVED** [review]

**Comment by itdependsnetworks** [comment]
Feel free to open another PR, but rather get infrahub merged in first before going back and forth. Overall I would prefer to keep things consistent, otherwise each I am left to defend the thought process.

**Pull request merged**